docs/DeveloperGuide.md

```diff
@@ -257,13 +257,13 @@ _{Explain here how the data archiving feature will be implemented}_
 
 **Target user profile**:
```

Comment:
Is it possible to add a little more details here about the target user? I.e. computing students who want to track study locations.